sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -150,7 +149,7 @@ This example lists all the secrets in the specified Key Vault.
 ```
 
 ### Async operations
-Python’s [asyncio package](https://pypi.org/project/asyncio/) (introduced in Python 3.4) and its two keywords `async` and `await` serves to declare, build, execute, and manage asynchronous code.
+Python’s [asyncio package](https://pypi.org/project/asyncio/) and its two keywords `async` and `await` serves to declare, build, execute, and manage asynchronous code.
```

Comment:
asyncio is in [the standard library](https://docs.python.org/3/library/asyncio.html)